oak_runtime/src/node/http/server.rs

```diff
@@ -358,6 +362,21 @@ impl HttpRequestHandler {
     }
 }
 
+/// Check if the request contains a `TRANSFER_ENCODING` header, and reject the request in that case
+/// by returning an error.
+fn validate_request(req: &HttpRequest) -> anyhow::Result<()> {
+    if req
```

Comment:
Just checking: are we not using HTTP2 only? It seems that's not affected: https://github.com/hyperium/hyper/security/advisories/GHSA-6hfq-h8hq-87mf

Reply:
In HttpServerNode we are using HTTP/1.1.